sdk/storage/azure-storage-file-share/tests/perfstress/download.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _ShareTest
+
+
+class DownloadTest(_ShareTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "downloadtest"
+        self.sharefile_client = self.share_client.get_file_client(file_name)
+        self.async_sharefile_client = self.async_share_client.get_file_client(file_name)
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = b'a' * self.args.size
```

Comment:
We should probably use random bytes instead of `aaa...`, for one thing to ensure it is non-compressable.  We already have a `random_stream` helper class, could we use it here?  We may also want a `random_array` helper class/method for scenarios which do need an array.

Reply:
Done!